Constitution.txt

```diff
@@ -1,490 +1,264 @@
+6: The right of the people to be secure in their persons, houses, papers, other information storage devices, and effects, against unwarranted searches and seizures, shall not be violated, and no Warrants shall issue, but upon probable cause, supported by Oath or affirmation, and particularly describing the place to be searched, and the persons or things to be seized, that are approved and signed by a duly appointed Judge or Justice of the Courts. All Warrants shall be matters of public record, made available to the public on or before the date of search or seizure.
 
-2: The Congress shall have power to enforce this article by appropriate legislation.   ratified #23
+7: No person shall be held to answer for a capital, or otherwise infamous crime, unless on a presentment or indictment of a Grand Jury, except in cases arising in the naval forces, or in the Militia, when in actual service in time of War or public danger; nor shall any person be subject for the same offense to be twice put in jeopardy of life, limb, liberty, or property; nor shall be compelled in any criminal case to be a witness against himself, nor be deprived of life, liberty, or property, without due process of law; nor shall private property be taken for public use, without just compensation.
 
-Amendment XXIV (Amendment 24 - Abolition of the Poll Tax Qualification in Federal Elections)
-1. The right of citizens of the United States to vote in any primary or other election for President or Vice President, for electors for President or Vice President, or for Senator or Representative in Congress, shall not be denied or abridged by the United States or any state by reason of failure to pay any poll tax or other tax.
+8: In all criminal prosecutions, the accused shall enjoy the right to a speedy and public trial, by an impartial jury of the State and district wherein the crime shall have been committed, which district shall have been previously ascertained by law, and to be informed of the nature and cause of the accusation; to be confronted with the witnesses against him; to have compulsory process for obtaining witnesses in his favor, and to have the Assistance of Counsel for his defense, such Counsel to be provided for by the State and district in which he is accused, in the event that he cannot procure Counsel for himself.
 
-2. The Congress shall have power to enforce this article by appropriate legislation.   ratified #24
+9: In Suits at common law, where the value in controversy shall exceed the value of five days' labor of any involved Party, the right of trial by jury shall be preserved, and no fact tried by a jury, shall be otherwise re-examined in any Court of these United States, than according to the rules of the common law.
```

Comment:
The original text of this Amendment laid out a specific dollar amount. Since the Congress no longer has authority over currency, I instead peg the amount to income levels for involved parties.